frontend/packages/knative-plugin/src/topology/knative-topology-utils.ts

```diff
@@ -208,7 +208,9 @@ export const getTriggerFilters = (resource: K8sResourceKind) => {
 };
 
 export const getKnativeDynamicResources = (
-  resources: TopologyDataResources,
+  resources:
+    | TopologyDataResources
+    | { [key: string]: K8sResourceKind & { data?: K8sResourceKind[] } },
```

Comment:
wrt to my previous comment on having `TopologyDataResources` type in AddPage, we can revert this changes.